src/modules/navigator/mission_block.cpp

```diff
@@ -452,13 +452,15 @@ MissionBlock::issue_command(const mission_item_s &item)
 		vcmd.param2 = item.params[1];
 		vcmd.param3 = item.params[2];
 		vcmd.param4 = item.params[3];
-		vcmd.param5 = item.params[4];
-		vcmd.param6 = item.params[5];
 
 		if (item.nav_cmd == NAV_CMD_DO_SET_ROI_LOCATION && item.altitude_is_relative) {
+			vcmd.param5 = item.lat;
```

Comment:
I thought here we lose the accuracy yet again but I was wrong.
https://github.com/PX4/Firmware/blob/18abeb1118f3db8e4012d290e1b8dc5d0883333a/msg/vehicle_command.msg#L105-L106

Reply:
Right, those two params are the exception. Apparently they are meant for LAT/LON storage :)

Reply:
Yep.